File: C#/1052.cs
using System; 

class URI {

    static void Main(string[] args) { 


        string[] meses = new string[3] {"January", "February", "March", "April",
                                        "May", "June", "July", "August",
                                        "September", "October", "November", "December"
                                        };


        int mes = Convert.ToInt32(Console.ReadLine());


        Console.WriteLine(meses[mes-1]);




    }
}

File: JS/URI/1052.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');


var mes = parseInt(lines);

var meses = ["January", "February", "March", "April",
    "May", "June", "July", "August",
    "September", "October", "November", "December"];

console.log(meses[mes-1]);